charts/trento-server/charts/trento-web/values.yaml

```diff
@@ -31,6 +31,16 @@ oidc:
   clientSecret: ""
   baseUrl: ""
 
+oauth2:
+  enabled: false
+  clientId: ""
+  clientSecret: ""
+  baseUrl: ""
+  authorizeUrl: ""
+  tokenUrl: ""
+  userUrl: ""
+  scopes: "profile email"
```

Comment:
Do we need a scopes value?

Reply:
Yes. If I don't set the default value (the same we have in the runtime), as helm will always populate this value (even with an empty string), this would be the passed value, and sending an empty string doesn't look appropriate

Reply:
Ah okay i wondered because we had here an empty string https://github.com/trento-project/ansible/commit/3bd8cf265c41f59734f1a040e5c745f9fc7f4433#diff-018072a0a2fd65ea8a5deb35db17eb24ec9297cc2fffd3ec7979e6cf402577e7R37. 
Thanks for clarification!

Reply:
I will add the default actually in the ansible code.
thanks!